lib/ssz_types/historical_batch_minimal.ex

```diff
@@ -0,0 +1,20 @@
+defmodule SszTypes.HistoricalBatchMinimal do
```

Comment:
should we have one file "historical_batch" per type and add the two modules there?

Reply:
Yeah, I think that would be better.